report/src/sections/01-introduction.tex

```diff
@@ -51,21 +51,49 @@ \subsection{Non-terminating Analysis}\label{subsec:non-terminating-analysis}
 \subsection{Contributions}\label{subsec:contributions}
 \todo[inline]{Casper says: These section (\ref{subsec:contributions}, \ref{subsec:related-work}, \ref{subsec:article-overview}) read like notes, I assume they are.}
 
-Use Halder concretely for abstract interpretation
+%Use Halder concretely for abstract interpretation:
+This paper builds on the work of abstract interpretation of database queries by Cortesi and Halder.
+Concretely, we use their work to analyze a database schema concerning a program using an alteration of the syntax they describe.
 
-This paper builds on work of abstract interpretation of database queries by Cortesi and Halder.
-Concretely, we use their work for analysing a database schema in relation to a program using the syntax they describe.
+%Improve on the levels of abstraction, i.e., add not just abstract values but also abstract tuples:
 
+Also, by building upon their work, we extend the level of abstraction used in the analysis.
+Where they only abstracted over values, this paper also abstracts over tuples in the database, ultimately guaranteeing a finite analysis.
+This is thoroughly covered in \autoref{sec:theory}.
 
-Improve on the levels of abstraction, i.e., add not just abstract values but also abstract tuples
+%Describe an implementable algorithm for performing static analysis targeting specific properties, given a program and database:
 
-Describe an implementable algorithm for performing static analysis targeting specific properties, given a program and database.
+Also included in the contribution is an implementable algorithm, which can be used for performing static analysis.
+This algorithm targets specific properties when given a program and its associated database.
+The allowed properties can be seen in the following \autoref{subsubsec:properties}.
 
-Prove soundness for the analysis.
+%Prove soundness for the analysis:
+To ensure that our abstractions are correct, we informally prove their soundness.
+By doing this, we can always be sure that our abstractions are an overestimation of the concrete values and tuples.
+This ensures that an abstract value always represents a concrete value.
```

Comment:
This is confusing the abstractions with the analysis. We prove that the analysis is sound, that is to say we prove that the analysis assignment that the analysis produces is an overestimation (see preliminaries).